fastplotlib/graphics/_base.py

```diff
@@ -116,6 +119,74 @@ def link(
             callback: callable = None,
             bidirectional: bool = False
     ):
+        """
+        Link this graphic to another graphic upon an ``event_type`` to change the ``feature``
+        of a ``target`` graphic.
+
+        Parameters
+        ----------
+        event_type: str
+            can be a pygfx event ("key_down", "key_up","pointer_down", "pointer_move", "pointer_up",
+            "pointer_enter", "pointer_leave", "click", "double_click", "wheel", "close", "resize")
+            or appropriate feature event (ex. colors, data, etc.) associated with the graphic (can use
+            ``graphic_instance.feature_events`` to get a tuple of the valid feature events for the
+            graphic)
+        target: Any
+            graphic to be linked to
+        feature: str
+            feature (ex. colors, data, etc.) of the target graphic that will change following
+            the event
+        new_data: Any
+            appropriate data that will be changed in the feature of the target graphic after
+            the event occurs
+        callback: callable, optional
+            user-specified callable that will handle event,
+            the callable must take the following four arguments
+            | ''source'' - this graphic instance
+            | ''target'' - the graphic to be changed following the event
+            | ''event'' - the ''pygfx event'' or ''feature event'' that occurs
+            | ''new_data'' - the appropriate data of the ''target'' that will be changed
+        bidirectional: bool, default False
+            if True, the target graphic is also linked back to this graphic instance using the
+            same arguments
+            For example:
+            .. code-block::python
+
+        Returns
+        -------
+        None
+
+        Examples
+        --------
+        .. code-block:: python
+
```

Comment:
let's just remove this example until it's implemented for `LineGraphic`.

Reply:
Keep it saved somewhere.

Reply:
Actually yes remove the `Examples` section entirely for `link()`, we should later just make a user guide that has a section on interaction. You can do the user guide later in a few weeks.